libraries/lambda_handlers/convert_handler.py

```diff
@@ -36,4 +42,22 @@ def _handle(self, event, context):
         converter = self.converter_class(source=source, resource=resource, cdn_file=cdn_file, options=options)
         results = converter.run()
         converter.close()  # do cleanup after run
+        if callback is not None:
```

Comment:
We should continue to try to keep any logic outside of the handler function as this is simply to be here to interface between the API Gateway and the actual tX code, just knowing how to deal with the event object to get tX the information it needs. I would think the callback would be actually part of the converter, not the handler to start the converter. This also helps in the tests as for all lambda_hanlder tests, it should simply mock the tX function ran (return None or whatever is returned by the API Gateway) and not test any other functionality.

So suggesting al l callback code be moved to converter.py so that if we ever do move off of Lambda, we don't lose this code.